libbeat/template/processor.go

```diff
@@ -488,5 +488,17 @@ func getDefaultProperties(f *mapping.Field) common.MapStr {
 	if f.CopyTo != "" {
 		properties["copy_to"] = f.CopyTo
 	}
+
+	if f.MetricType != "" || f.Unit != "" {
+		meta := common.MapStr{}
+		if f.MetricType != "" {
+			meta["metric_type"] = f.MetricType
```

Comment:
There is an ongoing discussion about changing these final meta names (to something like `_es_metric_type`). That said, I think our `fields.yml` definitions should stay the same and we would only need to change it here. I will push to make sure we have a decision before freeze

Reply:
Thanks! Yes, I saw it after I wrote this PR. Glad I steered away from having meta specified directly :)

Reply:
also I just went looking and saw you did it in the exact same way for integrations :smile: